app/src/Book.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: academy
+ * Date: 15/11/2017
+ * Time: 09:39
+ */
+
+namespace App;
+/**
+ * Book is a DB entity, with a constructor to create itself. The constructor has a default of null and can be passed an individual book id.
+ * @package App
+ */
+class Book
+{
+    public $db;
+    public $id;
+    public $title;
+    public $price;
+    public $description;
+    public $image;
+
+    /**
+     * Book constructor.
+     * @param int $id (book id)
+     * @param PDO $db (database connection)
+     */
+    public function __construct(\PDO $db = null, int $id = null)
+    {
+        if (isset($db)) {
```

Comment:
can technically exclude this if as $db defaults to null so you would be setting $this->db to null, which is fine